### DISCUSSION TITLE
How to get progress of the video

### DISCUSSION THREAD

**Discussion by amit0021** [jhomlala/betterplayer#653] [Q&A] [answered]
I've checked the add event section. But unable to understand how to put into the project. Actually I want to listen the video progress and show ad at some progress like 10%,20% of the video.

```
_betterPlayerController.addEventsListener((event) {
      if (event.betterPlayerEventType == BetterPlayerEventType.progress) {

        print("Better player event: ${event.betterPlayerEventType}");
      }
    });
```

This is the code I've used don't know how to put logic since not getting the progress numbers.

**Comment by jhomlala** [comment, marked as answer]
Just use ` _betterPlayerController.videoPlayerController.position`.
